src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -30,6 +31,80 @@ public bool HasAdditionalAssemblies
+                switch (Model.LoadState.State)
+                {
+                    case PackageLoadState.StateTypes.Unloaded: return Resources.PackageStateUnloaded;
+                    case PackageLoadState.StateTypes.Loaded: return Resources.PackageStateLoaded;
+                    case PackageLoadState.StateTypes.Error: return Resources.PackageStateError;
+                    default:
+                        return "Unkonwn package state";
+                }
+            }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageLoadStateTooltip
+        {
+            get
+            {
+                switch (Model.LoadState.ScheduledState)
+                {
+                    case PackageLoadState.ScheduledTypes.ScheduledForLoad: return Resources.PackageStateScheduledForLoadTooltip;
+                    case PackageLoadState.ScheduledTypes.ScheduledForUnload: return Resources.PackageStateScheduledForUnloadTooltip;
+                    case PackageLoadState.ScheduledTypes.ScheduledForDeletion: return Resources.PackageStateScheduledForDeletionTooltip;
+                    default:
+                        break;
+                }
+
+                switch (Model.LoadState.State)
+                {
+                    case PackageLoadState.StateTypes.Unloaded: return Resources.PackageStateUnloadedTooltip;
+                    case PackageLoadState.StateTypes.Loaded: return Resources.PackageStateLoadedTooltip;
+                    case PackageLoadState.StateTypes.Error: return string.Format(Resources.PackageStateErrorTooltip, Model.LoadState.ErrorMessage);
+                    default:
+                        return "Unkonwn package state";
+                }
+            }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageViewContextMenuUninstallText
+        {
+            get { return Model.BuiltInPackage ? Resources.PackageContextMenuUnloadPackageText : Resources.PackageContextMenuDeletePackageText; }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageViewContextMenuUninstallTooltip
+        {
+            get { return Model.BuiltInPackage ? Resources.PackageContextMenuUnloadPackageTooltip : Resources.PackageContextMenuDeletePackageTooltip; }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageViewContextMenuUnmarkUninstallText
+        {
+            get { return Model.BuiltInPackage ? Resources.PackageContextMenuUnmarkUnloadPackageText : Resources.PackageContextMenuUnmarkDeletePackageText; }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageViewContextMenuUnmarkUninstallTooltip
+        {
+            get { return Model.BuiltInPackage ? Resources.PackageContextMenuUnmarkUnloadPackageTooltip : Resources.PackageContextMenuUnmarkDeletePackageTooltip; }
+        }
```

Comment:
I did not use a converter (instead of these public properties) because it did not seem to simplify the situation very much